README.md

```diff
@@ -2,6 +2,57 @@
 
 Core database, replication, swarming, and chat APIs for cabal.
 
+## Table of Contents
+
+<details><summary>Click to expand</summary>
```

Comment:
Note that the table of contents is automatically generated by `hallmark` when doing `hallmark --fix`. So if any entries are added to the readme they should be automatically updated when the changelog is updated.